infrastructure/dynamo_db.tf

```diff
@@ -134,6 +134,38 @@ module "zip_store_reference_dynamodb_table" {
   owner       = var.owner
 }
 
+module "stitch_store_reference_dynamodb_table" {
```

Comment:
Can we change this name also, as there's isn't a `Stitch Store` bucket. Maybe 'LloydGeorgeStitchJobMetadata'?